recipes/botan/all/conanfile.py

```diff
@@ -35,6 +49,20 @@ class BotanConan(ConanFile):
                        'with_sqlite3': False,
                        'with_zlib': False,
                        'with_boost': False,
+                       'with_sse2': True,
+                       'with_ssse3': True,
+                       'with_sse4_1': True,
+                       'with_sse4_2': True,
+                       'with_avx2': True,
+                       'with_bmi2': True,
+                       'with_rdrand': True,
+                       'with_rdseed': True,
+                       'with_aes_ni': True,
+                       'with_sha_ni': True,
+                       'with_altivec': True,
```

Comment:
And removed on non-arm architectures.

Reply:
same as above

Reply:
THe problem is not about if botan will use or not, each different value will result in a different package id. So, if consider that botan won't use a flag, but it's available as option, so the same package can have different package ids. I agree with @madebr, they should be removed, because they affect the package id.